### DISCUSSION TITLE
MTV Classic

### DISCUSSION THREAD

**Discussion by ElusiveTheorist** [iptv-org/community#808] [Q&A]
### Channel Name

MTV Classic

### Channel ID

MTVClassicEast.us

### Notes

Hello!  Please add the American channel MTV Classic. All the broadcasts that I found are not available for some reason.

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/us_moveonjoy.m3u#L44-L47
